app/views/bus_stops/_form.haml

```diff
@@ -6,37 +6,34 @@
     Completed at #{format_datetime stop.completed_at}
 = form_for stop, url: bus_stop_path(stop.hastus_id) do |f|
   %table.edit-form
-    %th
-      Stop Info
+    %thead
+      %tr
+        %th
+          Stop Info
+    %tbody
       %tr
         %td= f.label :name
         %td= stop.name
       %tr
         %td= f.label :hastus_id, 'Hastus ID'
         %td= stop.hastus_id
-      - if current_user.admin?
```

Comment:
I do not know why we would wish to only show the route list to admins. Potentially a holdover from when it was editable?